tests/test_flake8_trio.py

```diff
@@ -146,7 +141,7 @@ def test_eval(test: str, path: str):
     assert parsed_args, "no parsed_args"
     assert expected, f"failed to parse any errors in file {path}"
 
-    plugin = read_file(path)
+    plugin = Plugin.from_filename(path)
```

Comment:
With `Plugin.from_filename` taking a `Path`, and `path` containing the full path, we don't need the helper function at all here.